### PR TITLE
[vi] update json and validation-inline

### PR DESCRIPTION
**Pull request by l3lackheart**

### CODE REVIEW

File: src/vi/validation-inline.php [hunk 1]
```diff
@@ -13,111 +12,111 @@
...
-    'file'           => 'The content must be a file.',
-    'filled'         => 'This field must have a value.',
+    'boolean'        => 'Trường này phải là true hoặc false.',
+    'confirmed'      => 'Giá trị xác nhận trong trường này không khớp.',
+    'date'           => 'Trường này không phải là định dạng của ngày-tháng.',
+    'date_equals'    => 'Trường này phải là một ngày bằng với :date.',
+    'date_format'    => 'Trường này không giống với định dạng :format.',
+    'different'      => 'Trường này và :other phải khác nhau.',
+    'digits'         => 'Độ dài của trường này phải gồm :digits chữ số.',
+    'digits_between' => 'Độ dài của trường này phải nằm trong khoảng :min and :max chữ số.',
```
Comment (caouecs) [inline src/vi/validation-inline.php:38]: and

**Review by caouecs — CHANGES_REQUESTED** [review]
Hello,
thank you, but just a typo on "and"

File: src/vi/validation-inline.php (hunk 1 above)
Comment (l3lackheart) [reply src/vi/validation-inline.php]: fixed! tks

**Comment by caouecs** [comment]
Thank you

**Pull request merged**